dev-python/elasticsearch-py/elasticsearch-py-5.5.2.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} pypy )
+
+ES_VERSION="5.6.7"
```

Comment:
Ok, now I'm going to be picky but there's a lot of useless whitespace here, and having empty line every line doesn't make reading easier. There's no reason for empty line between `ES_VERSION` and `PYTHON_COMPAT`.